azure-pipelines.yml

```diff
@@ -32,8 +32,8 @@ jobs:
         displayName: 'Install dependencies'
       - script: npm run lint
         displayName: 'Run lint'
-      # - script: npm run security
-      #   displayName: 'Run NPM audit'
+      - script: npm audit --production
+        displayName: 'Run NPM audit'
```

Comment:
Why only production? In theory dev dependencies can be vulnerable too, but at the moment we’re not auditing anything at all, so let's merge

Reply:
right, but currently we can't fix yargs-parser issue. https://github.com/webpack/webpack-dev-server/pull/2636#issuecomment-638170801